app/(default)/webstories/page.tsx

```diff
@@ -1,20 +1,31 @@
-import React from "react";
+"use client";
+import React, { useState } from "react";
 import StoriesCards from "@/components/webstories/StoriesCards";
-import { DataFiles } from "../../../components/utils/data"; // Import Data from data file
-import Header from "@/components/ui/header";
-const page = () => {
+import { DataFiles } from "@/components/utils/data";
+import Header from "@/components/webstories/components/Header";
+import RootLayout from "@/app/layout";
+
+const Page = () => {
   const Data = DataFiles.map((item) => ({
```

Comment:
this can be defined outside React component, since we dont need it to be computed at every render

Reply:
ok done!!